### ISSUE TITLE
Chapter 14 - Train/test/validation set wrong variable names

### ISSUE THREAD

**Issue by AhmedOmran0** [ageron/handson-ml2#261]
Under "Pretrained models for transfer learning", the flowers dataset is split up using:

```
test_set_raw, valid_set_raw, train_set_raw = tfds.load("tf_flowers",
    split=["train[:10%]", "train[10%:25%]", "train[25%:]"],
    as_supervised=True)`
```

But the preprocessing function and batching is applied to (so far undefined) names `train_set`, `valid_set` and `test_set`:

```
batch_size = 32
train_set = train_set.shuffle(1000)
train_set = train_set.map(preprocess).batch(batch_size).prefetch(1)
valid_set = valid_set.map(preprocess).batch(batch_size).prefetch(1)
test_set = test_set.map(preprocess).batch(batch_size).prefetch(1)
```

If I'm not mistaken, this should read 
```
batch_size = 32
train_set = train_set_raw.shuffle(1000)
train_set = train_set.map(preprocess).batch(batch_size).prefetch(1)
valid_set = valid_set_raw.map(preprocess).batch(batch_size).prefetch(1)
test_set = test_set_raw.map(preprocess).batch(batch_size).prefetch(1)
```

-A

**Comment by UPstartDeveloper** [comment]
Hello @AhmedOmran0 I just checked [in cell 53 of the Ch.14 notebook](https://github.com/ageron/handson-ml2/blob/master/14_deep_computer_vision_with_cnns.ipynb) for that code snippet - but I think someone must have fixed that issue already, it matches the code you have above 😄

**Comment by ageron** [comment]
Thanks @AhmedOmran0 and @UPstartDeveloper , indeed I must have fixed this a while back.
Closing this issue.
